packages/fxa-payments-server/server/lib/routes/post-metrics.js

```diff
@@ -3,7 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const joi = require('joi');
-const { amplitude } = require('../amplitude');
+const { amplitude, getCountryCode, getLocation } = require('../amplitude');
+// const { eventGroupNames } = require('../../../src/lib/amplitude');
```

Comment:
Not sure what this line is about -- can it be removed?

Reply:
yes, done